### ISSUE TITLE
Failing on travis

### ISSUE THREAD

**Issue by mlubin** [JuliaOpt/SCS.jl#26]
The current release/master has been failing on travis in a number of mysterious ways:

https://travis-ci.org/JuliaOpt/JuMP.jl/jobs/64000258#L873
```
while loading /Users/travis/build/JuliaOpt/JuMP.jl/test/model.jl, in expression starting on line 179
     - With solver SCS.SCSSolver
dyld: lazy symbol binding failed: Symbol not found: _validateLinSys
  Referenced from: /Users/travis/.julia/v0.4/Homebrew/deps/usr/lib/libscsdir64.dylib
  Expected in: flat namespace
dyld: Symbol not found: _validateLinSys
  Referenced from: /Users/travis/.julia/v0.4/Homebrew/deps/usr/lib/libscsdir64.dylib
  Expected in: flat namespace
```

https://travis-ci.org/JuliaOpt/SCS.jl/jobs/55384836#L268
```
ERROR: LoadError: LoadError: OverflowError()
 in hcat at abstractarray.jl:572
 in feasible_exponential_conic at /home/travis/.julia/v0.4/SCS/test/direct.jl:73
```

And on #25:
https://travis-ci.org/JuliaOpt/SCS.jl/jobs/62869361#L346
```
----------------------------------------------------------------------------
	SCS v1.0.7 - Splitting Conic Solver
	(c) Brendan O'Donoghue, Stanford University, 2012
----------------------------------------------------------------------------
Lin-sys: sparse-direct, nnz in A = 8545
eps = 1.00e-04, alpha = 1.80, max_iters = 20000, normalize = 1, scale = 5.00
Variables n = 173, constraints m = 520
Cones:	primal zero / dual free vars: 100
	linear vars: 150
	soc vars: 165, soc blks: 12
	sd vars: 75, sd blks: 3
	exp vars: 15, dual exp vars: 15
Setup time: 6.40e-03s
----------------------------------------------------------------------------
 Iter | pri res | dua res | rel gap | pri obj | dua obj | kap/tau | time (s)
----------------------------------------------------------------------------
     0|      inf       inf      -nan      -inf       inf       inf  6.79e-04 
   100|      inf       inf      -nan       inf       inf       inf  2.43e-02 
   160|      inf       inf      -nan       inf       inf       inf  3.79e-02 
----------------------------------------------------------------------------
Status: Infeasible
Timing: Total solve time: 3.79e-02s
	Lin-sys: nnz in L factor: 24110, avg solve time: 1.21e-04s
	Cones: avg projection time: 1.03e-04s
----------------------------------------------------------------------------
Certificate of primal infeasibility:
|A'y|_2 * |b|_2 = 4.3364e-05
dist(y, K*) = 0
b'y = -1.0000
============================================================================
ERROR: LoadError: LoadError: AssertionError: sol.ret_val == 1
```

**Comment by IainNZ** [comment]
Seems kinda busted on `pkg.julialang.org` too

**Comment by joehuchette** [comment]
I've been hitting this for a while now on OSX. Since this seems like an upstream issue, any thoughts @bodono? Should we open a ticket upstream?

**Comment by joehuchette** [comment]
(Specifically the ``_validateLinSys`` issue).

**Comment by bodono** [comment]
I can't replicate the `_validateLinSys` issue, is it correctly linking to the library? SCS.jl was updated recently, does that fix it? The `OverflowError()` is a known julia bug: https://github.com/JuliaLang/julia/issues/11320

**Comment by joehuchette** [comment]
I'm wondering if it's a problem with Homebrew(.jl?), actually. Has the bottle been updated since the latest changes to the SDP format?

**Comment by mlubin** [comment]
Can't we just work around the ``OverflowError`` by reading in the data from a file?

**Comment by bodono** [comment]
@joehuchette The bottle hasn't been updated as far as I know, it should be updated for the new API.

@mlubin This PR: https://github.com/JuliaOpt/SCS.jl/pull/28 makes the example smaller, so the `OverflowError` is no longer a problem, but it's failing too with a really strange error that's either my lack of understanding of julia, or perhaps another julia bug.

The nightly julia linux test is failing, but the release one is passing. The one that's failing has
```
ERROR: LoadError: UndefVarError: nidx not defined
 in orderconesforscs at /home/travis/.julia/v0.4/SCS/src/SCSSolverInterface.jl:253
```
but that line is just
```
252             nidx = length(idxs)
253             A = [A; -sparse(1:nidx, idxs, ones(nidx), nidx, num_vars)]
```

Any ideas?

**Comment by madeleineudell** [comment]
@bodono what's the right version of SCS to bottle? do you have a release or
tag for it? we'd like to get SCS working with Convex.jl in time for the
workshop we're running at JuliaCon this Saturday.

On Wed, Jun 24, 2015 at 2:47 AM, bodono <notifications@github.com> wrote:

> @joehuchette <https://github.com/joehuchette> The bottle hasn't been
> updated as far as I know, it should be updated for the new API.
>
> @mlubin <https://github.com/mlubin> This PR: #28
> <https://github.com/JuliaOpt/SCS.jl/pull/28> makes the example smaller,
> so the OverflowError is no longer a problem, but it's failing too with a
> really strange error that's either my lack of understanding of julia, or
> perhaps another julia bug.
>
> The nightly julia linux test is failing, but the release one is passing.
> The one that's failing has
>
> ERROR: LoadError: UndefVarError: nidx not defined
>  in orderconesforscs at /home/travis/.julia/v0.4/SCS/src/SCSSolverInterface.jl:253
>
> but that line is just
>
> 252             nidx = length(idxs)
> 253             A = [A; -sparse(1:nidx, idxs, ones(nidx), nidx, num_vars)]
>
> Any ideas?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaOpt/SCS.jl/issues/26#issuecomment-114809135>.
>



-- 
Madeleine Udell
www.stanford.edu/~udell

**Comment by bodono** [comment]
I just release v1.1.5, that can be bottled up.

**Comment by bodono** [comment]
Once it's bottled we have to update SCS.jl to download the right version.

**Comment by madeleineudell** [comment]
@tkelman could you build SCS v1.1.5 for windows?

**Comment by tkelman** [comment]
@madeleineudell I sure can.#29 - let appveyor take a look at it before merging though

**Comment by madeleineudell** [comment]
closed by #29
